staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics.go

```diff
@@ -39,6 +39,15 @@ var (
 		[]string{"resource"},
 	)
 
+	eventsCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "watch_cache_events_total",
+			Help:           "Counter of events dispatched in watchcache broken by resource type.",
```

Comment:
nit: some help texts are writing watchcache as `watchcache` and others as `watch cache`, could you make that consistent?

Reply:
done